pipeline/cloud/schemas/runs.py

```diff
@@ -265,14 +265,16 @@ class ContainerRunError(BaseModel):
 
 class ContainerRunCreate(BaseModel):
     # run_id is optional since it's just used for attaching logs to a run
-    run_id: t.Optional[str]
-    inputs: t.List[RunInput]
+    run_id: str | None
+    inputs: list[RunInput]
+    async_run: bool = False
+    callback_url: str | None
```

Comment:
Is this gonna be a pre-signed url type of thing for the container to publish result to main api or something?

Reply:
yes exactly